packages/shared/src/lib/numberFormat.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { getPrice, removeNonNumber } from './numberFormat';
+import {
+  dummyPaddleProductItemDollar,
+  dummyPaddleProductItemYen,
+} from '../../__tests__/fixture/paddle';
+
+describe('function removeNonNumber', () => {
+  it('should remove all non-number characters from a string', () => {
+    const value = '$1,234.56';
+    const result = removeNonNumber(value);
+    expect(result).toBe('1234.56');
+  });
+
+  it('should return the same value if there is no non-numeric character', () => {
```

Comment:
Maybe also try tests with the 2 extremes.
So something where money is worthless (zambia maybe?)
and with super high currency value

Reply:
I don't think the currency should really matter since we can only work with the format that Paddle gives us. Even if it is going to be from an extreme level of currency, if the formatting of the data returned to us is wrong, this would also be wrong. This is more of how we handle different formats.

Reply:
Similar to currency, the value wouldn't really be affected much as what we're trying to test is the format and whether the calculation ran. Anyway, I've added the tests. Let me know if you want anything to be adjusted.

Reply:
Maybe the way I set it up is not clear since the intricate details are within the fixtures. But yeah, the function is more about whether to run the previous calculation or not.